GenderPayGap.WebUI/Search/WordSplittingRegex.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Text.RegularExpressions;
+
+namespace GenderPayGap.WebUI.Search
+{
+    public static class WordSplittingRegex
+    {
+        private static Regex regex;
+
+        public static Regex Regex
+        {
+            get
+            {
+                if (regex != null)
+                {
+                    return regex;
+                }
+
+                // The "language=regexp" comments below tell Resharper to do regex syntax highlighting
+                // "CG" means "character group" - i.e. [A-Z] in regex
+
+                //language=regexp
+                string wordCG = "[a-zA-Z0-9'-]";

```

Comment:
Is a hyphen here counted as a word character then? So `co-op` would remain as `co-op`?

Reply:
It's hard to tell because a hyphen is used for `a-z` too

Reply:
Regex patterns are always a bit hard to parse

`[]` means a set of characters that includes `a-z`, `A-Z`, `0-9`, `'` and `-`
So `'` and `-` are part of the word.

The change I'm about to push removes punctuation *after* splitting into words.
So, `co-operative group` becomes [`cooperative`, `group`] rather than [`co`, `operative`, `group`]